src/Plugins/PHPBugs.php

```diff
@@ -0,0 +1,128 @@
+<?php  declare(strict_types=1);
+namespace Room11\Jeeves\Plugins;
```

Comment:
I realise this is copypasta from another file in Jeeves as there are several like this for some reason but can you unfuck this indentation please?

```php
<?php declare(strict_types=1);

namespace Room11\Jeeves\Plugins;
```